### ISSUE TITLE
Add buttons fail accessibility standard

### ISSUE THREAD

**Issue by discodavey** [libero/editor#502]
## Description
'+' symbol used in add buttons does not read in a useful way for screen readers. In addition text contrast on hover and focus is not sufficient to pass contrast standards.

## Tasks
- [ ] Replace '+' character with the word 'Add' on all add buttons
- [ ] Remove background hover and focus colour change leaving only pointer. 

## DoD
- [ ] Screen readers read out the button text so that the available action is understood. e.g. 'Add author'.
- [ ] No background state change on hover/focus.

**Comment by NickDuf** [comment]
@discodavey are my tasks accurate? the button element is being used.

**Comment by discodavey** [comment]
Agree, Did not see the button tag but I think there are too many span tags inside it.
